src/main/webapp/js/stats-loader.js

```diff
@@ -0,0 +1,23 @@
+// Fetch stats and display them in the page.
+function fetchStats(){
+    const url = '/stats';
+    fetch(url).then((response) => {
+      return response.json();
+    }).then((stats) => {
+      const statsContainer = document.getElementById('stats-container');
+      statsContainer.innerHTML = '';
+      const messageCountElement = buildStatElement('Message count: ' + stats.messageCount);
+      statsContainer.appendChild(messageCountElement);
+    });
+  }
+
+  function buildStatElement(statString){
+   const statElement = document.createElement('p');
+   statElement.appendChild(document.createTextNode(statString));
```

Comment:
While this formulation is fine, it is a little simpler to assign to the `textContent` property as in:

```
statElement.textContent = statstring;
```

https://developer.mozilla.org/en-US/docs/Web/API/Node/textContent